src/tools/tidy/src/style.rs

```diff
@@ -174,6 +174,11 @@ pub fn check(path: &Path, bad: &mut bool) {
 
         let can_contain =
             contents.contains("// ignore-tidy-") || contents.contains("# ignore-tidy-");
+        // Enable testing ICE's that require specific (untidy)
+        // file formats easily eg. `issue-1234-ignore-tidy.rs`
+        if filename.contains("ignore-tidy") {
+            return;
+        }
```

Comment:
This is a hack in the best meaning of the word.

Reply:
We could also place these kind of tests in a specific directory and filter those out?

Reply:
The filename opt out looks good to me.
It seems better to keep tests in directories where they actually belong to.

Reply:
cc @Mark-Simulacrum